### ISSUE TITLE
Does not work in Firefox?

### ISSUE THREAD

**Issue by Kuibagit** [MisterPhilip/x-forwarded-for#12]
Hello,

I noticed on firefox x-originating-ip, x-remote-ip, x-remote-addr headers are not working.

So I checked the code, found that the browser-polyfill.js file loaded here does not exist.
https://github.com/MisterPhilip/x-forwarded-for/blob/8dbd2ad852bdd0804c5e129e779d924e8d2a3587/platform/firefox/options.html#L162

I am not sure if this issue is causing the title to not work properly.

**Comment by MisterPhilip** [comment]
It seems to be related to https://github.com/MisterPhilip/x-forwarded-for/blob/master/src/options.js#L138, `ReferenceError: assignment to undeclared variable newSettings`

```js
            'saveHeaders': () => {
                console.log("save headers");
                newSettings = {
                    spoofIp: settings.spoofIp,
                    previous: settings.previous || [],
                    headers: el.headers.filter(checkbox => checkbox.checked && allowedHeaders.includes(checkbox.value)).map(checkbox => checkbox.value)
                };
                methods.setSettings(newSettings);
            }
```

Should be an easy fix

**Comment by MisterPhilip** [comment]
This is fixed in v0.6.0, which should be rolling out shortly. Thanks for the report, @Kuibagit!
